packages/graphback-cli/src/helpers/transformOpenApiSpec.ts

```diff
@@ -21,6 +21,12 @@ export const transformOpenApiSpec = async () => {
     checkDirectory(configInstance);
 
     const { folders, openApi } = configInstance.config;
+
+    if (!openApi) {
+        logError(`No OpenAPI configuration found in 'graphback.json' configuration file.`)
```

Comment:
We should be able to use defaults. 

```suggestion
        openApi = { includeComments: false, includeQueriesAndMutations: false }
```

Reply:
Alternative - thanks to JS being bad language:

```suggestion
        openApi = {}
```

Reply:
> We should be able to use defaults.

Defaults was another option I was thinking and it is a good way to go! Let's do this.

Reply:
> Alternative - thanks to JS being 'awesome' language:

Aah yeah, but since we have a `OpenApiConfig` with non-optional fields, I think `ts` won't like it.